### PR TITLE
fix: bump legacy-table-chart to 0.11.20

### PR DESCRIPTION
**Pull request by ktmud**
### CATEGORY

- [x ] Bug Fix

### SUMMARY

Minor fixes for the data table:

- [Fix scroll body height on very long column names and in very small containers](https://github.com/apache-superset/superset-ui-plugins/pull/398)
- [Adjust the position of the sort icons to align them at the bottom of table head](https://github.com/apache-superset/superset-ui-plugins/pull/399)

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

- Before
image

- After
image

### TEST PLAN

1. Create a table with a very long metric name
2. Put it into a small grid container in the dashboard
3. The table should not overflow out of the chart container.

### ADDITIONAL INFORMATION

N/A

### REVIEWERS

@etr2460 @graceguo-supercat @kristw

### CODE REVIEW

**Comment by etr2460** [comment]
Why are there so many package lock changes here?

**Comment by ktmud** [comment]
> Why are there so many package lock changes here?

Did a fresh `npm install` this time by removing both `node_modules` and `package-lock.json`. 

```bas
rm -rf node_modules package-lock.json && ENV="" NODE_ENV="development" npm install
```

Maybe not a good idea?

**Comment by kristw** [comment]
@ktmud That is kinda dangerous.

File: superset-frontend/package-lock.json [hunk 1]
```diff
@@ -7670,28 +7670,28 @@
           "dependencies": {
             "abbrev": {
               "version": "1.1.1",
-              "resolved": "",
+              "resolved": false,
```
Comment (ktmud) [inline superset-frontend/package-lock.json]: Empty strings become `false`s again. 🤦‍♂

Comment (etr2460) [reply superset-frontend/package-lock.json]: lol this i can deal with :P

**Review by etr2460 — APPROVED** [review]
lgtm, thanks!

**Pull request merged**